src/components/layout/SidebarList.tsx

```diff
@@ -137,6 +139,11 @@ const SidebarList = <Type extends unknown>({
         primary={item[titleKey]}
         secondary={formatDesc ? formatDesc(item[descKey]) : item[descKey]}
       />
+      {closedKey && !expiredItems.includes(item) && item[closedKey] && (
+        <ListItemIcon>
+          <WarningAmber color='warning' />
```

Comment:
Synes ikke det er nødvendig å legge til et slikt ikon i sidemenyen. Det er svært sjeldent at arrangementer er stengt, i tillegg er det ikke intuitivt at et varsel-symbol betyr at det er stengt. Det er også ikke helt heldig å legge inn en closed-prop som i realiteten er der kun for arrangementsadmin når sidebaren også brukes av andre admin-sider.

Reply:
Enig!